spoc/io.py

```diff
@@ -4,7 +4,10 @@
 from typing import Dict, Union
 import pandas as pd
 import dask.dataframe as dd
-from .models import FragmentSchema, AnnotatedFragmentSchema, HigherOrderContactSchema
+from .contacts import Contacts
+from .pixels import Pixels
+from .dataframe_models import FragmentSchema, ContactSchema
+from .fragments import Fragments
 
 
 class FileManager:
```

Comment:
It could be sensible to add some options to `.read_parquet` in the future, such as using pyarrow dtypes with pandas 2.0.*

Reply:
Yes, I would add those as an parameter to FileManager